ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -196,13 +199,13 @@ data:
       pipeline_stages:
       - cri: {}
       - match:
-          selector: '{app="event-exporter", namespace="openshift-e2e-loki"}'
-          stages:
-          - static_labels:
-              type: kube-event
+          selector: '{namespace!="openshift-e2e-loki", app!="event-exporter"}'
+          action: drop
```

Comment:
Cleaning this up as I discovered it has potential to double log things in other namespaces in weird ways, I added a static pod in default ns that just logged Hello world every 5s. This config was picking it up. Now we drop everything that isn't in namespace openshift-e2e-loki, and isn't app event exporter. (as the promtail pods are in here too, we want those logged by the above)